src/api/commons/queries/names.elastic.js

```diff
@@ -0,0 +1,19 @@
+import metas from './metas.query';
+
+export default [
+  ...metas,
+  {
+    $project: {
+      _id: 0,
+      officialName: { $ifNull: ['$usualName', null] },
```

Comment:
I think there is no need for ifNull in that case. If it does not exists you don't want it to be indexed anyway.
Usually i use it to return the value in the api response even if it's missing.

Reply:
Fixed